### PR TITLE
Repo Clean up

### PR DESCRIPTION
**Pull request by anajuliabit**
## Changes
- [x] Fix pre commit hook to ignore deleted files 
- [x] Remediate compiler warnings
- [x] Add linter to CI
- [x] Remove unused workflows

### Docs

- [x] Reorder docs files
- [x] Contributing file
- [x] Guideline file
- [x] Governance walk-through
- [x] Mips walk-through
- [x] Governance contracts 
- [x] Readme
- [x] TOC 

## Deleted

- [x] CrossChainJsonProposal contract and docs
- [x] proposals folder (inside root)
- [x] Timelock proposal
- [x] 4626 docs and contracts deleted
- [x] Delete deprecated folder post multichain governor upgrade
- [x] Outstanding unused branches

### CODE REVIEW

**Comment by anajuliabit** [comment]
@ElliotFriedman will remediate this last requested in the new branch I'm working on

File: src/governance/IArtemisGovernor.sol [hunk 1]
```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: BSD-3-Clause
+pragma solidity 0.8.19;
+
+import {IWormhole} from "@protocol/wormhole/IWormhole.sol";
+
+/// @notice interface for the Artemis Governor Contract
+interface IArtemisGovernor {
```
Comment (prateek105) [inline src/governance/IArtemisGovernor.sol:7]: This interface is not being used anywhere. Interface in interfaces folder is used. we can delete this.

**Review by prateek105 — DISMISSED** [review]
lgtm

File: yarn.lock [hunk 2]
```diff
@@ -26,19 +26,19 @@
...
   version "1.0.2"
-  resolved "https://registry.yarnpkg.com/@pnpm/network.ca-file/-/network.ca-file-1.0.2.tgz#2ab05e09c1af0cdf2fcf5035bea1484e222f7983"
+  resolved "https://registry.npmjs.org/@pnpm/network.ca-file/-/network.ca-file-1.0.2.tgz"
   integrity sha512-YcPQ8a0jwYU9bTdJDpXjMi7Brhkr1mXsXrUJvjqM2mQDgkRiz8jFaQGOdaLxgjtUfQgZhKy/O3cG/YwmgKaxLA==
   dependencies:
     graceful-fs "4.2.10"
 
 "@pnpm/npm-conf@^2.1.0":
   version "2.2.2"
-  resolved "https://registry.yarnpkg.com/@pnpm/npm-conf/-/npm-conf-2.2.2.tgz#0058baf1c26cbb63a828f0193795401684ac86f0"
```
[the remark this answers is not in the document]
Comment (anajuliabit) [reply yarn.lock]: ran npm i instead of yarn by mistake. Fixed

**Pull request merged**